project1/src/components/Buttons/styles/FavoriteButtonSlugStyle.ts

```diff
@@ -0,0 +1,35 @@
+import { createStyles } from "@material-ui/core/styles";
+
+export const styles = () =>
+  createStyles({
+    root: {
+      float: "right",
```

Comment:
this shouldn't be part of the button style
it makes the button not reusable in other situation where the float is not right